extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -274,8 +278,15 @@ export class ShareWidget {
    */
   copyUrlToClipboard_() {
     const url = Services.documentInfoForDoc(this.getAmpDoc_()).canonicalUrl;
-
-    if (!copyTextToClipboard(this.win, url)) {
+    const element = this.win.document.querySelector('amp-story');
+    const shareText =
+      this.storeService_.get('title') +
+      ' by ' +
+      element.getAttribute('publisher') +
+      ' via #WebStories ' +
+      url;
```

Comment:
Good call on changing the text here, but the UI really says `Get Link` so I think keeping it to just the URL without additional sharing text would make more sense.